packages/deployments/contracts/contracts/core/messaging/connectors/Connector.sol

```diff
@@ -40,9 +41,9 @@ abstract contract Connector is ProposedOwnable, MerkleTreeManager, ConnectorMana
 
   event Process(bytes32 leaf, bool success, bytes returnData);
 
-  event MessageSent(bytes data, address caller);
+  event RootUpdateSent(bytes data, address caller);
 
-  event MessageProcessed(address from, bytes data, address caller);
+  event RootUpdateReceived(address from, bytes data, address caller);
```

Comment:
do we want to be more specific with outbound / aggregate roots in the events if we have functions that are more opinionated?

Reply:
as in, with the naming here?

Like, `OutboundRootSent` or `AggregateRootReceived` ?